ReactAndroid/src/main/third-party/android/support/v4/BUCK

```diff
@@ -57,60 +134,227 @@ fb_native.android_prebuilt_aar(
 )
 
 fb_native.android_prebuilt_aar(
-    name = "lib-support-v4-support-fragment",
-    aar = ":lib-support-v4-support-fragment-aar",
+    name = "support-fragment",
+    aar = ":support-fragment-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "livedata-core",
+    aar = ":livedata-core-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "livedata",
+    aar = ":livedata-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "loader",
+    aar = ":loader-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "localbroadcastmanager",
+    aar = ":localbroadcastmanager-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "print",
+    aar = ":print-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "slidingpanelayout",
+    aar = ":slidingpanelayout-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "swiperefreshlayout",
+    aar = ":swiperefreshlayout-aar",
 )
 
+fb_native.android_prebuilt_aar(
+    name = "versionedparcelable",
+    aar = ":versionedparcelable-aar",
+)
+
+fb_native.android_prebuilt_aar(
+    name = "viewpager",
+    aar = ":viewpager-aar",
+)
+
+# remote files
 fb_native.remote_file(
-    name = "android-lifecycle-runtime-aar",
-    sha1 = "95428e5f6bf6875a6a1125d2157c3836a1a837f5",
-    url = "mvn:android.arch.lifecycle:runtime:aar:1.1.0",
+    name = "asynclayoutinflater-aar",
+    sha1 = "3ae7643d120e6da3adbe2d698de923f48c904d1f",
+    url = "mvn:com.android.support:asynclayoutinflater:aar:28.0.0",
 )
 
```

Comment:
can you add?

fb_native.remote_file(
    name = "support-annotations-binary.jar",
    sha1 = "ed73f5337a002d1fd24339d5fb08c2c9d9ca60d8",
    url = "mvn:com.android.support:support-annotations:jar:28.0.0",
)